spec/features/edit_album_spec.rb

```diff
@@ -48,10 +48,10 @@
       expect(page).to have_content "New name"
       expect(page).to have_content "New description"
       within('#photo0') do
-        expect(find('img')['src']).not_to have_content "Airturbine.jpg"
+        expect(find('img')['src']).to have_content "agriculture.jpg"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.